### DISCUSSION TITLE
is there a way to show venv/pyev/conda env outside of python segment?

### DISCUSSION THREAD

**Discussion by simpleranchero** [JanDeDobbeleer/oh-my-posh#5316] [Troubleshoot]
It works fine for python segment. but I want to know my env even outside of python project folders. what are my options?

**Comment by lewis-yeung** [comment]
You can set the `display_mode` property to `always`. See the [documentation][docs].

[docs]: https://ohmyposh.dev/docs/segments/languages/python#properties
